release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -176,6 +176,11 @@ Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plug-in, where
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-12-nw-ipv6-spoke-cluster-support"]
+==== Support for IPv6 spoke clusters from dual-stack hub clusters
```

Comment:
```suggestion
==== Support for deploying/provisioning IPv6 spoke clusters from dual-stack hub clusters
```
Minor wording suggestion

Reply:
perhaps "provisioning" is the correct choice as that's what is mentioned in the first sentence. Thanks